lecture_2/homework/app/models/melee.rb

```diff
@@ -0,0 +1,8 @@
+class Melee < Weapon
+  belongs_to :warrior, class_name: "Hussar", class_name: "Samurai"
```

Comment:
Pomysł ciekawy, ale niestety nie zadziała w taki sposób. `belongs_to` to nic innego jak zwykła metoda w rubym. A co się stanie jeśli do metody przekażemy ten sam argument dwa razy? Druga wartość zastąpi wcześniejszą. Wobec tego tę broń można przydzielić tylko samurajom 😢